### ISSUE TITLE
 Error: Invalid CSV Playlist - First row must be a header containing the columns TITLE and ARTIST. 

### ISSUE THREAD

**Issue by abolition123** [NBS-LLC/ng-spotify-importer#221]
Hi,

I have exported the shazaam csv and tried to import it but i get the error from the title.

 Error: Invalid CSV Playlist - First row must be a header containing the columns TITLE and ARTIST. 

What to do?

Cheers!

**Comment by NBS-LLC** [comment]
Please attach a sample of the CSV. I'll take a look :)

**Comment by Bruno-Fabijanic** [comment]
Hi,

Thanks for te quick reply 👍

Please find attached.


Kind regards,

Bruno Fabijanić

**Comment by NBS-LLC** [comment]
I don't think you can reply with an attachment via email.

See if replying directly via GH works. Otherwise please inline the first 10 rows of the CSV.

**Comment by gregorybelhumeur** [comment]
I ran into the same issue.

Here's the top rows of my CSV :

```
TITLE, ARTIST
"Les eaux de mars", "Stacey Kent"
"The Look of Love", "Diana Krall"
"Sympathique", "Pink Martini"
"La Javanaise", "Madeleine Peyroux"
"Don't Know Why", "Norah Jones"
```

**Comment by NBS-LLC** [comment]
@gregorybelhumeur thanks for the sample. Looks like I have a bug in how the header is parsed.

As a workaround just remove the space so it becomes: `TITLE,ARTIST`

**Comment by NBS-LLC** [comment]
Fixed in v1.4.3 which has been deployed to production. Closing.
